### PR TITLE
Diff between merge_issue and latest beta

### PR DESCRIPTION
**Pull request by bassosimone**
Please, no more rebases!

### CODE REVIEW

File: ooniprobe.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1154,7 +1309,7 @@
 				STRIP_INSTALLED_PRODUCT = NO;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				USER_HEADER_SEARCH_PATHS = "";
-				VALID_ARCHS = "arm64 armv7s i386 x84_64";
+				VALID_ARCHS = "armv7s armv7 i386 x86_64";
```
Comment (bassosimone) [inline ooniprobe.xcodeproj/project.pbxproj]: ^^^ @lorenzoPrimi

File: ooniprobe.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -1203,13 +1355,22 @@
 				STRIP_INSTALLED_PRODUCT = NO;
 				TARGETED_DEVICE_FAMILY = "1,2";
 				USER_HEADER_SEARCH_PATHS = "";
-				VALID_ARCHS = "arm64 armv7s i386 x84_64";
+				VALID_ARCHS = "armv7s armv7";
```
Comment (bassosimone) [inline ooniprobe.xcodeproj/project.pbxproj]: ^^^ @lorenzoPrimi

File: ooniprobe/View/TabsViewController.m [hunk 3]
```diff
@@ -11,18 +11,6 @@ - (void)viewDidLoad{
...
-    tabItem.title = NSLocalizedString(@"Dashboard.Tab.Label", nil);
-
-    UITabBarItem * tabItem1 = [self.tabBar.items objectAtIndex: 1];
-    tabItem1.image = [UIImage imageNamed:@"tab_test_results"];
-    tabItem1.title = NSLocalizedString(@"TestResults.Overview.Tab.Label", nil);
-
-    UITabBarItem * tabItem2 = [self.tabBar.items objectAtIndex: 2];
-    tabItem2.image = [UIImage imageNamed:@"tab_settings"];
-    tabItem2.title = NSLocalizedString(@"Settings.Title", nil);
 }
```
Comment (bassosimone) [inline ooniprobe/View/TabsViewController.m]: ^^^ @lorenzoPrimi

File: ooniprobe.xcodeproj/project.pbxproj (hunk 2 above)
Comment (lorenzoPrimi) [reply ooniprobe.xcodeproj/project.pbxproj]: good question. What are che VALID_ARCHS now?

**Comment by bassosimone** [comment]
Can be closed because #213 looks good. (This PR was openned to discuss changes between different branches after a rebase brought us back of about one month; see #213)

**Pull request closed without merging**